test/WebJobs.Extensions.DurableTask.Analyzers.Test/ActivityFunction/NameAnalyzerTests.cs

```diff
@@ -102,6 +105,23 @@ public static string SayHelloTupleOnContext([ActivityTrigger] IDurableActivityCo
             string name = context.GetInput<(string, int)>();
             return $""Hello {name}!"";
         }
+
+        [FunctionName(nameof(SayHelloByMethodName))]
+        public static string SayHelloByMethodName([ActivityTrigger] IDurableActivityContext context)
+        {
+            string name = context.GetInput<string>();
+            return $""Hello {name}!"";
+        }
+    }
+
+    public class SayHelloByClassName
+    {
+        [FunctionName(nameof(SayHelloByClassName))]
```

Comment:
Lets make one of these attributes just a string, so we can test that it works in mismatch scenarios.